fury/actor/actor.py

```diff
@@ -0,0 +1,54 @@
+import fury.primitive as fp
+import numpy as np
+from fury.v2.actor.materials import _create_mesh_material
+from fury.v2.actor.geometry import buffer_to_geometry, create_mesh
+
+
+def sphere(
+    centers,
+    colors,
+    *,
+    radii=1.0,
+    phi=16,
+    theta=16,
+    vertices=None,
+    faces=None,
+    opacity=1,
+    material='phong',
+    enable_picking=True
+):
+
+
+    scales = radii
+    directions = (1, 0, 0)
+
+
+    if faces is None and vertices is None:
+        vertices, faces = fp.prim_sphere(phi=phi, theta=theta)
```

Comment:
what happens if only vertices is None ? will it work?

Reply:
> what happens if only vertices is None ? will it work?

@skoudoro No, It will not work and I can't imagine any use case for giving the vertices and faces of a sphere to create a sphere. Do you think we should remove it or keep it?